packages/runtime/src/helpers/functions.ts

```diff
@@ -364,58 +369,154 @@ const changeExtension = (file: string, extension: string) => {
   return join(dirname(file), base + extension)
 }
 
-const getSSRDependencies = async (publish: string): Promise<string[]> => {
+const getPrerenderManifest = async (publish: string) => {
   const prerenderManifest: PrerenderManifest = await readJSON(join(publish, 'prerender-manifest.json'))
 
-  return [
-    ...Object.entries(prerenderManifest.routes).flatMap(([route, ssgRoute]) => {
-      if (ssgRoute.initialRevalidateSeconds === false) {
-        return []
-      }
+  return prerenderManifest
+}
+
+type CachedPageValue = Extract<ResponseCacheValue, { kind: 'PAGE' }> | { html: object | string }
```

Comment:
`ResponseCacheValue` is an exported union type from Next.js, however it doesn't export the types in the union. I use extract to get the actual type I need.